hstream/src/HStream/Server/Handler/Query.hs

```diff
@@ -84,10 +84,19 @@ executeQueryHandler sc@ServerContext {..} (ServerNormalRequest _metadata Command
       let sources = snd <$> inNodesWithStreams
           sink    = snd outNodeWithStream
           query   = P.ViewQuery (textToCBytes <$> sources) (CB.pack . T.unpack $ sink) schema
-      create (transToStreamName sink)
-      (qid,_) <- handleCreateAsSelect sc plan commandQueryStmtText query
-      atomicModifyIORef' P.groupbyStores (\hm -> (HM.insert sink accumulation hm, ()))
-      returnCommandQueryResp (mkVectorStruct (cBytesToText qid) "view_query_id")
+      -- make sure source streams exist
+      existedStreams <- V.toList <$> Core.listStreamNames sc
+      existedViews   <-              Core.listViewNames   sc
+      let existSources = existedStreams <> existedViews
+      case L.find (`notElem` existSources) sources of
```

Comment:
It does similar work to [here](https://github.com/hstreamdb/hstream/blob/3fd95ec642642b0708fe59978bb4f392227c9888/hstream/src/HStream/Server/Handler/Query.hs#L211). I am not sure which is better but to keep the consistency, it is better to use the same mechanism.

Reply:
thx fixed [ae633ae](https://github.com/hstreamdb/hstream/pull/1014/commits/ae633aecf6a3739063787682f8dcdca5991b56c8)